scripts/ops/safeAllocation.ts

```diff
@@ -0,0 +1,97 @@
+import fs from "fs-extra";
+import { ethers } from "hardhat";
+import Safe from "@gnosis.pm/safe-core-sdk";
+import EthersAdapter from "@gnosis.pm/safe-ethers-lib";
+import { SafeEthersSigner, SafeService } from "@gnosis.pm/safe-ethers-adapters";
+import { Wallet } from "ethers";
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { chefV2Address, ops } from "../constants";
+
+type AllocationConfig = {
+  PoolId: number;
+  Allocation: number;
+  Rewarder: string;
+  LpToken: string;
+};
+
+const { SAFE_SIGNER_MNEMONIC = undefined, AURORA_API_KEY } = process.env;
+if (!AURORA_API_KEY) {
+  throw new Error("*** AURORA_API_KEY NOT FOUND IN ENV");
+}
+
+const AURORA_URL = "https://mainnet.aurora.dev/" + AURORA_API_KEY;
+const SAFE_SERVICE_URL = "https://safe-transaction.aurora.gnosis.io/";
+const provider = new JsonRpcProvider(AURORA_URL);
+
+if (!SAFE_SIGNER_MNEMONIC) {
+  throw new Error("*** SAFE_SIGNER_MNEMONIC NOT FOUND IN ENV ***");
+}
+
```

Comment:
nit validate `env` vars before logic

```suggestion
if (!SAFE_SIGNER_MNEMONIC) {
  throw new Error("*** SAFE_SIGNER_MNEMONIC NOT FOUND IN ENV ***");
}

const AURORA_URL = "https://mainnet.aurora.dev/" + AURORA_API_KEY;
const SAFE_SERVICE_URL = "https://safe-transaction.aurora.gnosis.io/";
const provider = new JsonRpcProvider(AURORA_URL);

```